admin/RELEASING.md

```diff
@@ -0,0 +1,18 @@
+# How-to release `rustls-platform-verifier`
+
+This document records the steps to publish new versions of the crate since it requires non-trivial preperation and ordering
+that needs to be remembered due to the Android component's distribution.
+
+## Steps
+
+1. Update main crate'a version in `rustls-platform-verifier/Cargo.toml`, and in any additional places.
+2. If any non-test changes have been made to the `android` directory since the last release:
+    1. Update Android artifact version in `android-release-support/Cargo.toml`
+    2. Bump dependency version of the Android support crate in `rustls-platform-verifier/Cargo.toml` to match the new one
+    3. Commit version increase changes on the release branch
```

Comment:
Do you have any preferences about how to create/name the release branch? We use [rel-xxx](https://github.com/rustls/rustls/blob/main/RELEASING.md#maintenance-point-releases) in rustls/rustls and rustls/webpki fwiw

Reply:
I don't have any preference, no. In other open source repositories for 1P I use `release-xxx` and then delete it once the tag has been created.

Reply:
> I don't have any preference, no. 

Let's standardize on the Rustls approach then to have one less thing to map between repos.

> and then delete it once the tag has been created.

We've been leaving our release branches around for subsequent point releases to use, but I don't feel strongly.